int32 motorID
int32 position
int32 velocity